crates/autopilot/src/infra/persistence/dto/order.rs

```diff
@@ -281,8 +281,11 @@ pub enum FeePolicy {
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Quote {
+    #[serde_as(as = "HexOrDecimalU256")]
```

Comment:
🤦, the foot-gun fires again.

We should probably cherry pick this fix outside of this PR to avoid starting to write fee policies with quote amounts in hex.

Reply:
Fix already merged with one of the previous PRs, it's just my code is not updated to the newest `main`:
https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/infra/persistence/dto/order.rs#L284